test-support/helpers/actions.js

```diff
@@ -0,0 +1,23 @@
+import Ember from 'ember';
+import {
+  findInputByLabel,
+  findLabelByText
+} from '../helpers/finders';
+
+export function clickLink(linkText) {
+  return function() {
+    const linkSelector = `a:contains('${linkText}')`;
```

Comment:
I would pass the string to the click function directly, no need for the `const` before hand